test/distributed/org/apache/cassandra/distributed/impl/Instance.java

```diff
@@ -374,6 +375,8 @@ public void startup(ICluster cluster)
                 }
                 JVMStabilityInspector.replaceKiller(new InstanceKiller());
 
+                ActiveRepairService.instance.start();
```

Comment:
this enables IR

Reply:
we don't block repair behind a feature flag; this is a boolean saying 'yep, IR was started'

Reply:
From what I can see, we're starting a scheduled thread here:

```
consistent.local.start();
        ScheduledExecutors.optionalTasks.scheduleAtFixedRate(consistent.local::cleanup, 0,
                                                             LocalSessions.CLEANUP_INTERVAL,
                                                             TimeUnit.SECONDS);
```

It looks like it will get shut down together with scheduled executors, but I still think we should hide it behind a flag just like netwroking and gossip.

Reply:
@ifesdjeen can you explain why feature flags exist?  Maybe there is a mismatch between what I thought and your thinking; I honestly thought it was for performance to start a cluster.

What feels weird to me is that full, validation, and preview repair will work without the new flag, but incremental repair will require it.  So to me it feels off.